RA_Consoles.h

```diff
@@ -86,6 +86,10 @@ enum ConsoleID
     TI83 = 79,
     Uzebox = 80,
 
+    Hubs = 100,
+    Events = 101,
+    Standalones = 102,
```

Comment:
This project is meant for applications integrating directly with the DLL. There will never be an emulator telling the DLL that it's designed for Hubs or Events, and if I understand how the standalones will interface with the server, they shouldn't need DLL support either.

Having said that, I'm not opposed to exposing the values.

Reply:
Thanks for providing this clarification. I am happy to close this PR if you'd prefer.